# joint_state_controller.yaml

joint_state_controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 50  # Tần suất xuất bản (Hz)
  joints:
    - joint_1
    - joint_2
    - joint_3
    - joint_4
    - joint_5
    - joint_6
    - joint_7

# Cấu hình controller để điều khiển các khớp robot
cobot_controller:
  type: "effort_controllers/JointGroupEffortController"
  joints:
    - joint_1
    - joint_2
    - joint_3
    - joint_4
    - joint_5
    - joint_6
    - joint_7
  pid_gains:
    joint_1: { p: 600, d: 30, i: 0 }
    joint_2: { p: 600, d: 30, i: 0 }
    joint_3: { p: 600, d: 30, i: 0 }
    joint_4: { p: 600, d: 30, i: 0 }
    joint_5: { p: 250, d: 10, i: 0 }
    joint_6: { p: 150, d: 10, i: 0 }
    joint_7: { p: 50,  d: 5,  i: 0 }
